svcb/svcb/build.py

```diff
@@ -252,12 +252,19 @@ def generateCMakeDecls(benchmarkObjs, sourceRootDir, supportedArchitecture, depe
             declStr += "  {}\n".format(macroName)
         declStr += ")\n"
       # Emit compiler flags
-      declStr += "{indent}target_compile_options({target_name} PRIVATE ${{SVCOMP_STD_{lang_ver}}})\n".format(
-        indent=cmakeIndent,
-        target_name=targetName,
-        lang_ver= lang_ver,
-        arch = arch.upper()
-      )
+      if coverage:
```

Comment:
the gcov files will end up in two distinct places:
1. the compile time information will end up next to the object files
2. the runtime counters will be created relative to the working directory of the executable